sdk/eventhub/eventhubs-checkpointstore-blob/test/internal/parseIntOrThrow.spec.ts

```diff
@@ -0,0 +1,104 @@
+// Copyright (c) Microsoft Corporation. All rights reserved.
+// Licensed under the MIT License.
+
+import { parseIntOrThrow } from "../../src/blobCheckpointStore";
+import { EnvVarKeys, getEnvVars } from "../public/utils/testUtils";
+import { ContainerClient } from "@azure/storage-blob";
+import { Guid } from "guid-typescript";
+import { BlobCheckpointStore } from "../../src";
+import { EventHubConsumerClient } from "@azure/event-hubs";
+import chai from "chai";
+const should = chai.should();
+const env = getEnvVars();
+
+describe("Blob Checkpoint Store", function(): void {
+  const service = {
+    storageConnectionString: env[EnvVarKeys.STORAGE_CONNECTION_STRING]
+  };
+  let containerClient: ContainerClient;
+  before("validate environment", async function(): Promise<void> {
+    should.exist(
+      env[EnvVarKeys.STORAGE_CONNECTION_STRING],
+      "define STORAGE_CONNECTION_STRING in your environment before running integration tests."
+    );
+  });
+
+  beforeEach(async () => {
+    containerClient = new ContainerClient(
+      service.storageConnectionString,
+      `container-${Guid.create()}`
+    );
+    await containerClient.create();
+  });
+
+  afterEach(async () => {
+    await containerClient.delete();
+  });
+
+  it("listOwnership should return an empty array", async function(): Promise<void> {
```

Comment:
Is this being newly added? Why?

I don't see it in the deletions.

Reply:
See this - https://github.com/Azure/azure-sdk-for-js/pull/13028/files/844df4afad92bc08dd2b6be84d4818e02fa79d26#r568886137

Reply:
Okay, so the `parseIntOrThrow` was deleted.
The other two tests in this file are duplicated. Can you remove the duplications?

Reply:
Done @HarshaNalluru